pkg/v1/tkg/client/upgrade_addon.go

```diff
@@ -182,11 +182,6 @@ func (c *TkgClient) DoUpgradeAddon(regionalClusterClient clusterclient.Client, /
 			clusterClient = currentClusterClient
 		case "addons-management/kapp-controller":
 			crsDisabledAddon = true
-		case "addons-management/tanzu-addons-manager":
```

Comment:
Are we assuming case "packages/management-package" will update the addons manager package after the removal of case "addons-management/tanzu-addons-manager"?
Question: Will "packages/management-package" have multiple packages?

Reply:
yeah all the management packages will use this addon type